unicodetools/src/main/java/org/unicode/tools/emoji/GenerateEmoji.java

```diff
@@ -1533,9 +1533,9 @@ private static void showOrdering(
 
             outPlain.println(
                     "# labels.txt\n"
-                            + "# Copyright © 1991-2016 Unicode, Inc.\n"
+                            + "# Copyright © 1991-Present Unicode, Inc.\n"
```

Comment:
FYI -- I don't know which files this generates; I am surprised that this was stuck on 2016, and I am not aware of where such files might live. @macchiati @nedley might know.

Reply:
labels.txt is generated in the `internal` charts directory, presumably for ingestion by CLDR. I’m honestly not sure why it needs to state a copyright when another file used by CLDR (emoji-ordering-rules.txt) doesn’t have one.

Reply:
why `Present`? was that @annebright approved? 

`CldrUtility.getCopyrightShort()` will return

```java
"Copyright \u00A9 1991-"
                        + Calendar.getInstance().get(Calendar.YEAR)
                        + " Unicode, Inc.";
```

Reply:
and actually, `"<!--\n"+getCopyrightString()+"\n-->\n"` from that same class is what CLDR uses to output its whole file header

Reply:
As @nedley wrote, this one doesn't really matter, because it just generates internal files.

Reply:
changed to use `Default.getYear()` like elsewhere in these tools